solr/core/src/java/org/apache/solr/servlet/RequestRateLimiter.java

```diff
@@ -157,7 +162,7 @@ public void decrementRequest() {
       }
     }
 
-    public boolean isUsedPoolNull() {
+    public boolean isReleasable() {
       return usedPool == null;
```

Comment:
Should be != null, right? Check the condition wherever this is called too, might need to invert those.

Reply:
Fixed